test/unittests/commands/excel2json/test_properties.py

```diff
@@ -99,25 +99,20 @@ def test_do_property_excel_compliance_problems(self) -> None:
         )
         expected_msg = regex.escape(
             (
-                "There is a problem with the excel file: 'Test'\n\n"
-                "The column 'name' must have values in the row(s):\n"
-                "    - 9\n\n"
-                "The column 'super' must have values in the row(s):\n"
-                "    - 2\n"
-                "    - 4\n"
-                "    - 9\n\n"
-                "The column 'object' must have values in the row(s):\n"
-                "    - 5\n"
-                "    - 9\n\n"
-                "The column 'gui_element' must have values in the row(s):\n"
-                "    - 6\n"
-                "    - 8\n"
-                "    - 9\n\n"
-                "The column 'label' must have values in the row(s):\n"
-                "    - 3\n"
-                "    - 8\n\n"
-                "The column 'gui_attributes' must have values in the row(s):\n"
-                "    - 7"
+                "The Excel file: 'Test' contains the following problems:\n\n"
```

Comment:
AFAIK, there shouldn't be a colon at this place, because it would interrupt the flow of the sentence. Colons are separators between sentences, or between sub-sentences (Teilsätze), but should not be placed in the middle of a sentence

```suggestion
                "The Excel file 'Test' contains the following problems:\n\n"
```